azure-iot-device/azure/iot/device/common/pipeline/config.py

```diff
@@ -29,6 +32,7 @@ def __init__(
         websockets=False,
         cipher="",
         proxy_options=None,
+        keep_alive=None,
```

Comment:
>None [](start = 19, length = 4)

This should probably be the default keepalive instead of None, as I mentioned in another comment